provisor/utils.py

```diff
@@ -67,29 +67,22 @@ def validate_pubkey(value):
 
 
 def validate_username(value):
-    reserved_usernames = [
-        'about', 'abuse', 'main', 'data', 'example', 'jabber', 'legal', 'invite',
-        'copyright', 'contact', 'board', 'feedback', 'support', 'anonymous', 'index',
-        'inbox', 'payment', 'donate', 'calendar', 'dotfiles', 'billing', 'billings',
-        'images', 'media', 'policy', 'manage', 'messages', 'mobile', 'official',
-        'staging', 'development', 'staff', 'portal', 'forum', 'forums', 'pictures',
-        'photos', 'status', 'finger', 'private', 'press', 'user', 'users', 'username',
-        'usernames', 'sitemap', 'team', 'teams', 'account', 'accounts', 'chat', 'mail',
-        'email', 'admin', 'admins', 'administrator', 'administrators', 'postmaster',
-        'hostmaster', 'webmaster'
-    ]
+    from reserved import RESERVED_USERNAMES
 
     # Regexp must be kept in sync with
-    #  https://github.com/hashbang/hashbang.sh/blob/master/src/hashbang.sh#L178-191
+    #  https://github.com/hashbang/hashbang.sh/blob/master/src/hashbang.sh#L186-196
     if re.compile(r"^[a-z][a-z0-9]{,30}$").match(value) is None:
         raise ValueError('Username is invalid')
-    if value in reserved_usernames:
+
+    if value in RESERVED_USERNAMES:
         raise ValueError('Username is reserved')
+
     user_exists = True
```

Comment:
Why not include `value` in the error message?

Up to you; just a suggestion.

Reply:
Then I would need to uniformize with the stuff in `hashbang.sh`, and that's kinda out-of-scope for this PR anyhow.